apps/sso/serializers.py

```diff
@@ -0,0 +1,120 @@
+from oauth2_provider.generators import generate_client_secret
+from oauth2_provider.models import (
+    get_access_token_model,
+    get_application_model,
+    get_grant_model,
+    get_refresh_token_model,
+)
+from rest_framework import serializers
+
+from apps.sso.models import ApplicationConsent
+
+# TODO:
```

Comment:
I think you should sort this out before merging.

Reply:
You're right. Any ideas for name?

Reply:
Perhaps `ExternalAuthentication` as prefix?

Reply:
Went with `SSO`!